src/pages/workspace/card/issueNew/CardNameStep.tsx

```diff
@@ -44,12 +61,25 @@ function CardNameStep() {
             <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mv3]}>{translate('workspace.card.issueNewCard.giveItName')}</Text>
             <FormProvider
                 formID={ONYXKEYS.FORMS.ISSUE_NEW_EXPENSIFY_CARD_FORM}
+                // TODO: change the submitButtonText to 'common.confirm' when editing and navigate to ConfirmationStep
                 submitButtonText={translate('common.next')}
                 onSubmit={submit}
+                validate={validate}
                 style={[styles.mh5, styles.flexGrow1]}
+                enabledWhenOffline
             >
-                {/* TODO: the content will be created in https://github.com/Expensify/App/issues/44309 */}
-                <View />
+                <InputWrapper
+                    InputComponent={TextInput}
+                    inputID={INPUT_IDS.CARD_TITLE}
+                    label={translate('workspace.card.issueNewCard.cardName')}
+                    hint={translate('workspace.card.issueNewCard.giveItNameInstruction')}
+                    aria-label={translate('workspace.card.issueNewCard.cardName')}
+                    role={CONST.ROLE.PRESENTATION}
+                    // TODO: default value for card name
+                    defaultValue={issueNewCard?.data?.cardTitle}
+                    containerStyles={[styles.mb6]}
+                    autoFocus={!issueNewCard?.data?.cardTitle}
```

Comment:
I believe autoFocus should always be true even when editing.

On the other hand, could you consider using `FocusTrapForScreens` instead of autoFocus props

Reply:
But would it be ok designwise? Looks like this on web when true for all the cases
<img width="358" alt="Screenshot 2024-07-04 at 12 41 22" src="https://github.com/Expensify/App/assets/28020445/955a099c-eb23-40ad-8047-5f583f322708">

Reply:
Can we update the behaviour so it leaves the cursor in the amount field but it's not selecting the entire amount